Upgrade to Composable Kernel 7.2 (#396)
amd/rocm-examples: Libraries/ComposableKernel/gemm/grouped_gemm/README.md

## Description

> This example demonstrates how to perform a grouped GEMM operation with the CK Tile programming model. Multiple variants are provided:

Reviewer: ```suggestion
This example demonstrates how to perform a grouped GEMM operation usingCK Tile. Multiple variants are provided:
```
